CefSharp.Core/Internals/CefRegisterCdmCallbackWrapper.h

```diff
@@ -0,0 +1,65 @@
+// Copyright � 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "Stdafx.h"
+#include "CefWrapper.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        private class CefRegisterCdmCallbackWrapper : public CefRegisterCdmCallback
+        {
+        private:
+            gcroot<TaskCompletionSource<CdmRegistration^>^> _taskCompletionSource;
```

Comment:
Sorry to be a pain, I'm fairly new to the CefSharp codebase and not that familiar with other areas of it.

In order to call the underlying CefRegisterWidevineCdm it wants a CefRegisterCdmCallback (defined in the Cef codebase). Can you please point me to an example where a similar process takes place but using the interface as you described?

Or are you saying pass our own defined interface into the callback wrapper and use that to trigger back through to the caller? But still use the CefRegisterCdmCallback internally.

Again, sorry for the noobness!